front-end/src/components/Themes/themes.js

```diff
@@ -1,65 +1,169 @@
+			alignItems: 'center',
+			justifyContent: 'center',
+			marginTop: '100px'
+		},
+		titleColor: {
+			color: '#110044',
+			fontFamily: 'Montserrat',
+			fontWeight: '400'
+		},
+		textColor: {
+			color: '#454152', 
+			familyFont: 'Montserrat',
+			fontWeight: '200'
 		}
 	},
 	light: {
 		cvContainer: {
-  			width: '100%',
+  			width: '80%',
+  			border: '3px solid #C6ACAC',
   			padding: '20px',
-  			color: '#550400',
+  			color: '#FFFFFF',
   			'display': 'flex',
   			'flexDirection': 'column',
   			'justifyContent': 'center',
-  			'alignItems': 'center'
+  			'alignItems': 'center',
+  			'backgroundColor': '#FFFFFF'
   		},
 		cvContent:{
 			display: 'flex',
 			'flexDirection': 'row-reverse',
-  			padding:'30px'
+  			padding:'30px',
+  			marginTop: '100px',
+  			justifyContent: 'center',
+  			alignItems: 'center',
+  			width: '100%'
 		},
-		centerPiece: {
+		rightPiece: {
+			width: '50%',
 			'paddingLeft':'150px', 
-			'borderLeft':'1px solid #A8383B', 
-			'marginLeft':'150px'
+			'borderLeft':'2px #EBE6E3',
+			'borderStyle': 'dashed'
+		},
+		leftPiece: {
+			width: '50%',
+			'paddingLeft':'150px'
+		},
+		wrapper: {
+			display: 'flex',
+			'alignItems': 'center',
+			'justifyContent': 'center',
+			'backgroundColor': '#EBE6E3'
+		},
+		imageDiv: {
+			width: '100%',
+			display: 'flex',
+			flexDirection: 'column',
+			alignItems: 'center',
+			justifyContent: 'center',
+			marginTop: '100px'
+		},
+		titleColor: {
+			color: '#473C36',
+			familyFont: 'Montserrat',
+			fontWeight: '400'
+		},
+		textColor: {
+			color: '#7F7069',
+			familyFont: 'Montserrat',
+			fontWeight: '200'
 		}
 	},
 	dark:{
 		cvContainer: {
-  			width: '100%',
+  			width: '80%',
+  			border: '3px solid #C6ACAC',
   			padding: '20px',
-  			color: '#1A4A00',
+  			color: '#FFFFFF',
   			'display': 'flex',
   			'flexDirection': 'column',
   			'justifyContent': 'center',
-  			'alignItems': 'center'
+  			'alignItems': 'center',
+  			'backgroundColor': '#FFFFFF'
   		},
 		cvContent:{
 			display: 'flex',
 			'flexDirection': 'row-reverse',
-  			padding:'30px'
+  			padding:'30px',
+  			marginTop: '100px',
+  			justifyContent: 'center',
+  			alignItems: 'center',
+  			width: '100%'
 		},
-		centerPiece: {
+		rightPiece: {
+			width: '50%',
 			'paddingLeft':'150px', 
-			'borderLeft':'1px solid #49D000', 
-			'marginLeft':'150px'
+			'borderLeft':'1px solid #8D4A4A'
+		},
+		leftPiece: {
+			width: '50%',
+			'paddingLeft':'150px'
+		},
+		wrapper: {
+			display: 'flex',
+			'alignItems': 'center',
+			'justifyContent': 'center',
+			'backgroundColor': '#A26868'
+		},
+		imageDiv: {
+			width: '100%',
+			display: 'flex',
+			flexDirection: 'column',
+			alignItems: 'center',
+			justifyContent: 'center',
+			marginTop: '100px'
+		},
+		titleColor: {
+			color: '#730303',
+			familyFont: 'Montserrat',
+			fontWeight: '400'
+		},
+		textColor: {
+			color: '#8D4A4A',
+			familyFont: 'Montserrat',
+			fontWeight: '200'
```

Comment:
i like doing the themes this way

Reply:
I like it too! It took me a while to get it right, but your theme changer example helped me a lot.